src/index.css

```diff
@@ -1,102 +1,154 @@
-:root {
-  font-family: Inter, system-ui, Avenir, Helvetica, Arial, sans-serif;
-  line-height: 1.5;
-  font-weight: 400;
-  color: rgba(255, 255, 255, 0.87);
-  background-color: #242424;
-  font-synthesis: none;
-  text-rendering: optimizeLegibility;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  -webkit-text-size-adjust: 100%;
+* {
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
 }
 
-a {
-  font-weight: 500;
-  color: #646cff;
-  text-decoration: inherit;
+body {
+  background-color: rgb(236, 236, 236);
 }
 
-a:hover {
-  color: #535bf2;
+#root {
+  padding: 10% 7% 10% 7%;
 }
 
-body {
-  margin: 0;
-  display: flex;
-  place-items: center;
-  min-width: 320px;
-  min-height: 100vh;
+.wrapper {
+  background-color: rgb(248, 245, 245);
 }
 
 h1 {
-  font-size: 3.2em;
-  line-height: 1.1;
+  font-family: 'Courier New', Courier, monospace;
 }
 
-button {
-  border-radius: 8px;
-  border: 1px solid transparent;
-  padding: 0.6em 1.2em;
-  font-size: 1em;
-  font-weight: 500;
-  font-family: inherit;
-  background-color: #1a1a1a;
-  cursor: pointer;
-  transition: border-color 0.25s;
+.navbar {
+  padding: 1.5vw 5vw 1.5vw 5vw;
+  display: flex;
+  align-items: center;
+  border-bottom: 1px solid rgb(236, 236, 236);
+  background-color: white;
+  gap: 3vw;
 }
 
-button:hover {
-  border-color: #646cff;
+nav ul {
+  display: flex;
+  gap: 2vw;
 }
 
-button:focus,
-button:focus-visible {
-  outline: 4px auto -webkit-focus-ring-color;
+.book-item {
+  margin: 0 5vw 0 5vw;
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  background-color: white;
+  border: 1px solid rgb(236, 236, 236);
 }
 
-@media (prefers-color-scheme: light) {
-  :root {
-    color: #213547;
-    background-color: #fff;
-  }
-
-  a:hover {
-    color: #747bff;
-  }
+.book-info {
+  padding: 3vw 2vw 3vw 2vw;
+}
 
-  button {
-    background-color: #f9f9f9;
-  }
+.link-btn {
+  background: none;
+  border: none;
+  color: rgb(129, 129, 253);
 }
 
-nav > ul {
+.buttons {
```

Comment:
- [ ] Good job working on the styling but there are just a few things that you need to address. Please pay great attention to the sections highlighted.
- [ ] Please kindly ensure that your design matches the original design guidelines(font size, background-color, color, padding, margin, etc)

### Your Design
![image](https://github.com/Mar12358/bookstore/assets/43995830/8299bb92-e7a2-428f-b4f6-0a1e1d06ec22)

### Original Design
![image](https://github.com/Mar12358/bookstore/assets/43995830/d91eb2d7-7c21-4ef5-9679-2686664c4b15)